### ISSUE TITLE
Upgrade failed

### ISSUE THREAD

**Issue by Josue-T** [YunoHost-Apps/searx_ynh#60]
Hello, my upgrade failed.

Note that it's on an arm board.

Here is my log:
```
2020-04-10 15:07:52,214: DEBUG - + '[' -n '' ']'
2020-04-10 15:07:52,216: DEBUG - + unset backup_file_checksum
2020-04-10 15:07:52,218: DEBUG - + ynh_systemd_action --service_name=nginx --action=reload
2020-04-10 15:07:52,219: DEBUG - + local legacy_args=nalpte
2020-04-10 15:07:52,221: DEBUG - + args_array=([n]=service_name= [a]=action= [l]=line_match= [p]=log_path= [t]=timeout= [e]=length=)
2020-04-10 15:07:52,223: DEBUG - + declare -Ar args_array
2020-04-10 15:07:52,225: DEBUG - + local service_name
2020-04-10 15:07:52,227: DEBUG - + local action
2020-04-10 15:07:52,229: DEBUG - + local line_match
2020-04-10 15:07:52,230: DEBUG - + local length
2020-04-10 15:07:52,231: DEBUG - + local log_path
2020-04-10 15:07:52,232: DEBUG - + local timeout
2020-04-10 15:07:52,234: DEBUG - + ynh_handle_getopts_args --service_name=nginx --action=reload
2020-04-10 15:07:52,235: DEBUG - + set +x
2020-04-10 15:07:52,337: DEBUG - + local service_name=nginx
2020-04-10 15:07:52,342: DEBUG - + local action=reload
2020-04-10 15:07:52,348: DEBUG - + local log_path=/var/log/nginx/nginx.log
2020-04-10 15:07:52,353: DEBUG - + local length=20
2020-04-10 15:07:52,356: DEBUG - + local timeout=300
2020-04-10 15:07:52,357: DEBUG - + [[ -n '' ]]
2020-04-10 15:07:52,358: DEBUG - + '[' reload == reload ']'
2020-04-10 15:07:52,359: DEBUG - + action=reload-or-restart
2020-04-10 15:07:52,360: DEBUG - + systemctl reload-or-restart nginx
2020-04-10 15:07:54,373: DEBUG - + [[ -n '' ]]
2020-04-10 15:07:54,375: DEBUG - + ynh_script_progression '--message=Making sure dedicated system user exists...'
2020-04-10 15:07:54,376: DEBUG - + set +x
2020-04-10 15:07:54,378: DEBUG - + set +x
2020-04-10 15:07:54,379: DEBUG - + echo '! Helper used in legacy mode !'
2020-04-10 15:07:54,380: DEBUG - + set +x
2020-04-10 15:07:54,381: DEBUG - + echo '[#########+..........] > Making sure dedicated system user exists...'
2020-04-10 15:07:54,382: DEBUG - + set -x
2020-04-10 15:07:54,384: DEBUG - + ynh_system_user_create --username=searx
2020-04-10 15:07:54,385: DEBUG - + local legacy_args=uhs
2020-04-10 15:07:54,387: DEBUG - + args_array=([u]=username= [h]=home_dir= [s]=use_shell)
2020-04-10 15:07:54,388: DEBUG - + declare -Ar args_array
2020-04-10 15:07:54,389: DEBUG - + local username
2020-04-10 15:07:54,391: DEBUG - + local home_dir
2020-04-10 15:07:54,392: DEBUG - + local use_shell
2020-04-10 15:07:54,393: DEBUG - + ynh_handle_getopts_args --username=searx
2020-04-10 15:07:54,395: DEBUG - + set +x
2020-04-10 15:07:54,396: DEBUG - + use_shell=0
2020-04-10 15:07:54,397: DEBUG - + home_dir=
2020-04-10 15:07:54,398: DEBUG - + ynh_system_user_exists searx
2020-04-10 15:07:54,400: DEBUG - + local legacy_args=u
2020-04-10 15:07:54,401: DEBUG - + args_array=([u]=username=)
2020-04-10 15:07:54,402: DEBUG - + declare -Ar args_array
2020-04-10 15:07:54,403: DEBUG - + local username
2020-04-10 15:07:54,405: DEBUG - + ynh_handle_getopts_args searx
2020-04-10 15:07:54,406: DEBUG - + set +x
2020-04-10 15:07:54,407: DEBUG - + echo '! Helper used in legacy mode !'
2020-04-10 15:07:54,408: DEBUG - + set +x
2020-04-10 15:07:54,510: DEBUG - + getent passwd searx
2020-04-10 15:07:54,511: INFO - [#########+..........] > Making sure dedicated system user exists...
2020-04-10 15:07:54,514: DEBUG - + ynh_script_progression '--message=Upgrading Searx...' --weight=7
2020-04-10 15:07:54,516: DEBUG - + set +x
2020-04-10 15:07:54,516: DEBUG - + set +x
2020-04-10 15:07:54,517: DEBUG - + echo '! Helper used in legacy mode !'
2020-04-10 15:07:54,518: DEBUG - + set +x
2020-04-10 15:07:54,619: DEBUG - + echo '[##########++++......] > Upgrading Searx...'
2020-04-10 15:07:54,646: WARNING - rm: impossible de supprimer '/opt/yunohost/searx/lib/python2.7/site-packages/setuptools': Aucun fichier ou dossier de ce type
2020-04-10 15:07:54,648: INFO - [##########++++......] > Upgrading Searx...
2020-04-10 15:07:54,650: DEBUG - + set -x
2020-04-10 15:07:54,651: WARNING - rm: impossible de supprimer '/opt/yunohost/searx/lib/python2.7/site-packages/setuptools-*': Aucun fichier ou dossier de ce type
2020-04-10 15:07:54,652: DEBUG - + rm -r /opt/yunohost/searx/lib/python2.7/site-packages/setuptools '/opt/yunohost/searx/lib/python2.7/site-packages/setuptools-*'
2020-04-10 15:07:54,654: DEBUG - + ynh_exit_properly
2020-04-10 15:07:54,655: DEBUG - + local exit_code=1
2020-04-10 15:07:54,656: DEBUG - + '[' 1 -eq 0 ']'
2020-04-10 15:07:54,656: DEBUG - + trap '' EXIT
2020-04-10 15:07:54,657: DEBUG - + set +eu
2020-04-10 15:07:54,658: DEBUG - + sleep 0.5
2020-04-10 15:07:55,061: DEBUG - + type -t ynh_clean_setup
2020-04-10 15:07:55,069: WARNING - [ERR] Upgrade failed.
2020-04-10 15:07:55,072: DEBUG - + ynh_clean_setup
```

**Comment by maniackcrudelis** [comment]
Strange to fail on a 2 years old [commit](https://github.com/YunoHost-Apps/searx_ynh/commit/f59da0dcbc1f35f98fbe32001e0a695171328001)
To add a force here is no so easy, with a wildcard...

Meanwhile, could you just create that missing file ?

**Comment by Josue-T** [comment]
Should be fixed by #61
